src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -39,23 +33,19 @@ def __init__(self, parentLogger=None):
         """Standard Constructor"""
 
         try:
-            indexPrefix = CSGlobals.getSetup().lower()
-
             # Connecting to the ES cluster
-            super().__init__("WorkloadManagement/ElasticJobParametersDB", indexPrefix, parentLogger=parentLogger)
+            super().__init__("WorkloadManagement/ElasticJobParametersDB", "job_parameters", parentLogger=parentLogger)
```

Comment:
Would it be possible to let the indexPrefix configurable? 
Sometimes data centers providing OpenSearch service have some specific requirements about the naming of the index prefix.

Reply:
Sure. Done.